challenge.md

```diff
@@ -6,40 +6,28 @@ Write your answers inside this file, where it's indicated by the comments.
 
 1. We just forked a repo on GitHub.com and want to start working on it locally. What command do we use to do that?
 
-<!-- Answer Starts Here -->
-
-<!-- Answer Ends Here -->
+Copy the link for the clone and enter the following code:
+git clone yourclonedgit@somewhereongithub
 
 2. OK, we just wrote some code. What command could we use to see a list of all the changes that have been made since the last commit?
 
-<!-- Answer Starts Here -->
-
-<!-- Answer Ends Here -->
+We could use git status
 
 3. Oops - it looks like there was a change that we forgot to include in our last commit. How can we revise that commit and fix things?
 
-<!-- Answer Starts Here -->
-
-<!-- Answer Ends Here -->
+We could go back and say git commit --ammend
 
 4. It looks like there was a change on the original repo that we forked from. How could we grab those changes and incorporate them into our local repo?
 
-<!-- Answer Starts Here -->
-
-<!-- Answer Ends Here -->
+git fetch upstream
```

Comment:
Not quite. If you do `git fetch`, you'd also need to to `git merge` or `git rebase` (things we haven't covered yet) to incorporate the changes that you grabbed from the upstream. The answer I was looking for here is `git pull upstream` (possibly `git pull upstream master`).